src/components/PaymentInfoLine.vue

```diff
@@ -105,7 +105,15 @@ export default class PaymentInfoLine extends Vue {
         height: 3.375rem;
         margin-top: -0.25rem;
         margin-bottom: -0.125rem;
-        margin-right: 1rem;
+        margin-right: 0;
+    }
+
+    .account >>> .account-image {
```

Comment:
I think the deep selector is not necessary:
```suggestion
    .account-image {
```

Additionally, the `.account-image` rule should come before the `.identicon` rule, as it's also in front in the template.